spec/services/products_renderer_spec.rb

```diff
@@ -16,19 +16,23 @@
     let(:cakes_supplier) { create(:supplier_enterprise) }
     let!(:product_apples) {
       create(:product, name: "apples", primary_taxon_id: fruits.id,
-                       supplier_id: fruits_supplier.id)
+                       supplier_id: fruits_supplier.id, inherits_properties: true)
     }
     let!(:product_banana_bread) {
-      create(:product, name: "banana bread", primary_taxon_id: cakes.id,
-                       supplier_id: cakes_supplier.id)
+      create(:product, name: "banana bread", inherits_properties: true,
+                       variants: [
+                         create(:variant, supplier: cakes_supplier, primary_taxon: cakes),
+                         create(:variant, supplier: fruits_supplier, primary_taxon: cakes)
+                       ]
+            )
```

Comment:
I think this creates some flackyness because "banana bread" has 2 variants with different supplier. I believe the order of the variant matter.